lib/tool_shed/test/base/twilltestcase.py

```diff
@@ -60,7 +60,7 @@ def setUp(self):
         self.galaxy_url = f"http://{self.galaxy_host}:{self.galaxy_port}"
         self.shed_tool_data_table_conf = os.environ.get('TOOL_SHED_TEST_TOOL_DATA_TABLE_CONF')
         self.file_dir = os.environ.get('TOOL_SHED_TEST_FILE_DIR', None)
-        self.tool_data_path = os.environ.get('GALAXY_TEST_TOOL_DATA_PATH')
+        self.tool_data_path = os.environ.get('GALAXY_TEST_TOOL_DATA_PATH', 'test/functional/tool-data/')
```

Comment:
If that's necessary for tests to pass I think it likely alerts to the fact that this would break under other circumstances as well.

Reply:
Found no other way to get the functional test working. But you are right, since there are probably other tests using data tables it makes one wonder why they are working. 

I remember that the problem was that `GALAXY_ROOT/tool-data` was used if I did not change this. 

Maybe this is related to the other comment? 

Any suggestions how to continue. I'm happy to try anything :)

Reply:
This is gone now.